Dockerfile

```diff
@@ -80,6 +81,7 @@ RUN apt-get update && apt-get --yes --force-yes --no-install-recommends install
      linuxdoc-tools \ 
      lmodern \
      maven \ 
+     openjdk-8-jdk \
```

Comment:
We are using the `jenkins/jenkins:lts` image which is already based on `openjdk:8-jdk`  so a JDK is already installed. I think there is no need to extra include it here?

I just checked (the current dev image) and `java -version` gives:
```shell
openjdk version "1.8.0_212"
OpenJDK Runtime Environment (build 1.8.0_212-b04)
OpenJDK 64-Bit Server VM (build 25.212-b04, mixed mode)
```

Reply:
The image doesn't seem to have everything exactly where you'd expect it, so my guess is they went for a lightweight, custom install of the JDK. Or something like that...It might be possible to do the equivalent with some strategic symlinking, but just doing a proper install seemed to fix it.

Reply:
I checked again with only the addition of `ant-optional` and it's indeed enough. Since the extra installation of `openjdk-8-jdk` has no benefit I will remove it and merge that PR.